src/NUnitFramework/tests/Internal/CollectionTallyTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using NUnit.Framework.Internal;
```

Comment:
Please can you add the copyright notice to here and CollectionEquivalentConstraintResultTests? 🙂